app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/screens/restaurant/uimodel/RestaurantUiModel.kt

```diff
@@ -0,0 +1,46 @@
+package co.co.nimblehq.showcases.poly.place.ui.screens.restaurant.uimodel
+
+import android.graphics.Bitmap
+import android.location.Location
+import android.os.Parcelable
+import co.co.nimblehq.showcases.poly.place.domain.model.Restaurant
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class RestaurantUiModel(
+    val id: String,
+    val thumbnailImage: Bitmap?,
+    val name: String,
+    val address: String,
+    val distance: Float?
+) : Parcelable
+
+fun Restaurant.toRestaurantUiModel(currentLatitude: Double, currentLongitude: Double) =
```

Comment:
```suggestion
fun Restaurant.toRestaurantUiModel(
    currentLatitude: Double,
    currentLongitude: Double,
) = RestaurantUiModel(
```
Let's reformat it. 🙏🏻